### ISSUE TITLE
Update sandbox base uri

### ISSUE THREAD

**Issue by mtvbrianking** [mtvbrianking/laravel-mtn-momo#27]
From: 

https://ericssonbasicapi2.azure-api.net/

To:

https://sandbox.momodeveloper.mtn.com/

**Comment by mtvbrianking** [comment]
In the interim, users can update this via `.env`

```diff
- MOMO_API_BASE_URI="https://ericssonbasicapi2.azure-api.net/"

+ MOMO_API_BASE_URI="https://sandbox.momodeveloper.mtn.com/"
```

**Comment by mtvbrianking** [comment]
Addressed in commit [a544807](https://github.com/mtvbrianking/laravel-mtn-momo/commit/a544807ba8a12fbcd624fb008f0ade30aa76a7a4)
